config/packages/prod/monolog.yaml

```diff
@@ -22,3 +25,11 @@ monolog:
             handler: deprecation
             max_level: info
             channels: ["php"]
+        slack_errors:
+            type: slack
+            token: '%env(SLACK_TOKEN)%'
+            channel: 'dev-kit-logs'
```

Comment:
I assume you're not using the ci channel, because the logging level is debug ?

Reply:
It had to be a private channel, and also `ci` feels a bit vague / inappropriate. In a subsequent PR, we might want to rework the thing that posts to that channel so that it do so via Monolog, by using another channel.